UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -447,6 +475,18 @@ public class ListSpriteDataSOWithOrder
 		public SpriteOrder SpriteOrder;
 		public List<SpriteDataSO> Sprites = new List<SpriteDataSO>();
 	}
+	public class BodyPartModification : MonoBehaviour
```

Comment:
I think this should be on its own file

Reply:
Wait, Haven't I seen those same methods in BodyParts?

Reply:
to be fair never meant to be used in this place but is cleaner

Reply:
yeah, it transfers it to different moral behaviour so you don't overwrite the component every time you modify it so inheritance works properly

Reply:
*mono behaviour

Reply:
uhhhm would we benefit of making an abstract parent and make BodyPart and BodyPartModification inherit from it? Since they both have the same methods.

Reply:
why?

Reply:
It is a question. By having a class with the same method names alone I can feel a little scratchy and wonder if we're doing something wrong with the architecture

Reply:
well code structure is
living health behaviour  ->root body part container -> Bodypart -> BodyPartModifications
For the general handling you would go through Bodypart , and if it interacts with the custom logic of BodyPartModifications you just look up for it in the body part,

Reply:
I see